src/bindgen/processor/virtual_override.cr

```diff
@@ -364,7 +364,7 @@ module Bindgen
 
         rules = @db.get_or_add(cpp_struct.name)
         rules.graph_node = crystal_struct
-        rules.crystal_type = typer.qualified(crystal_struct.name, in_lib: true)
+        rules.crystal_type = crystal_struct.name
```

Comment:
Qualified name is not necessary here, because `copy_structure` already signals the fact that the jumptable struct will be defined under `lib Binding`.